binding.gyp

```diff
@@ -47,6 +47,16 @@
 						'MACOSX_DEPLOYMENT_TARGET': '10.11',
 						'GCC_ENABLE_CPP_EXCEPTIONS': 'YES'
 					},
+					'actions': [
+						{
+							'action_name': 'copy_mobiledevice',
+							'inputs': [ ],
```

Comment:
Move the variables into the inputs array including the output dir, then set the action's 2nd arg to `'<@(_inputs)'`. That should work with multiple args. Right!?

Reply:
They do!

Reply:
This doesn't seem to actually work, I believe that it's required for the values specified as `inputs` to exist. If they don't, then we get the failure seen here https://jenkins.appcelerator.org/blue/organizations/jenkins/cli%2Fnode-ios-device/detail/PR-63/8/pipeline